models/iaf_cond_alpha.h

```diff
@@ -66,12 +66,14 @@ Simple conductance based leaky integrate-and-fire neuron model
 Description
 +++++++++++
 
-iaf_cond_alpha is an implementation of a spiking neuron using IAF dynamics with
+``iaf_cond_alpha`` is an implementation of a spiking neuron using IAF dynamics with
 conductance-based synapses. Incoming spike events induce a postsynaptic change
 of conductance modelled by an alpha function. The alpha function
 is normalized such that an event of weight 1.0 results in a peak current of 1 nS
 at :math:`t = \tau_{syn}`.
```

Comment:
This does the `t = \tau_{syn}` whereas above I corrected to double back-tics. Should probably also been `:math:` above, right?